static/js/popovers.js

```diff
@@ -431,22 +431,26 @@ export function show_user_profile(user) {
         callback(name, key) {
             function hide_profile_tab() {
                 $("#profile-tab").hide();
-                $(".subscription-group-list").show();
             }
             switch (name) {
                 case "User groups":
                     hide_profile_tab();
                     $("#streams-tab").hide();
+                    $(".subscription-group-tab").show();
+                    $(".subscription-stream-tab").hide();
```

Comment:
I have added a comment about restructuring html, after that maybe these changes will not be required for both streams and groups case.